src/staticanalysis/bot/static_analysis_bot/coverity/coverity.py

```diff
@@ -202,6 +211,11 @@ def __init__(self, issue, revision):
         self.body = None
         self.nb_lines = 1
 
+        if settings.cov_full_stack:
+            # We cache the issue that is going to be parsed from 'events' in order to retrieve the full
+            # stack of the defect
```

Comment:
Couldn't you calculate the extended message here and save it in a property of self?

Reply:
sure!